apps/cli/commit_session.py

```
@@ -0,0 +1,177 @@
+        logger.error("Failed to init DB: %s", e)
+        return 1
+
+    targets: Set[int] = set()
+    if args.session_id is not None:
+        targets.add(int(args.session_id))
+
+    since = _normalize_run_started_at(args.run_started_at)
+    if since:
+        try:
+            window_sessions = db_find_in_progress_sessions(since=since)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)